### DISCUSSION TITLE
Implémentation d'une commande ai:help

### DISCUSSION THREAD

**Discussion by Izocel** [RvD-Projects/whatTheChuckBot#102] [Ideas]
Transfère d'issue:#100   
> @CtrlAiDel  Implémentation d'une commande ai:help afin de pouvoir donner une liste des commandes accessibles au user.
> 
> Je suggères que cette list d'aide ne soit qu'un fichier .json que tu puisses ou que l'ont puisse mettre a jour, afin de donner plus de détail sur les différents commandes ou fonction de ai:
> 
> example: ai:"nom" prompt - Utilisation du modèle par son nom complet (Ex: llava-llama3:latest) ai:"alias" prompt - Utilisation du modèle par son alias (Ex: llava-llama3) ai: prompt - Utilisation du modèle par défault(Actuellement llava-llama3) ai:list - Liste complète des modèles ai:stop - Je me sert de celle-ci mais je ne connais pas la description exacte ai:reset - Je me sert de celle-ci mais je ne connais pas la description exacte
> 
> Peut-être qu'en tapant ai:help, la liste des users admissibles pourraient être partager mais ceci est moins important que le reste je dirais.

**Pour les users je ne suis pas d'accord c'est confidentiel et ça doit le rester.**

**Pour la documentation, j'opterais plus vers un fichier markdown : `~/plugins/{name}/commands/help.md`
Comme ça le formatage est rapide, prêt pour Discord® et visionable dans un lecteur de markdown.**

**EXAMPLE-MARKDOWN:**

### Select a model by specifying its full name (e.g. llava-llama3:latest):
```
ai:llava-llama3 Can you help me with...
```

### Select a model by specifying its alias (Ex: llava-llama3):
```
ai:llava-llama3 Can you help me with...
```

### Use the last selected model: 
```
ai: Can you help me with...
```

### View the full list of models:
```
ai:list
```

### Display various information about the current context:
```
ai:conf
```

### Remove all messages from the context and select the default model:
```
ai:reset
```

### Display this help page:
```
ai:help
```

### Sélectionner un modèle en spécifiant son nom complet (Ex: llava-llama3:latest) :
```
ai:llava-llama3 Peux-tu m'aider avec...
```

### Sélectionner un modèle en spécifiant son alias (Ex: llava-llama3) :
```
ai:llava-llama3 Peux-tu m'aider avec...
```

### Utiliser le dernier modèle sélectionné : 
```
ai: Peux-tu m'aider avec...
```

### Afficher la liste complète des modèles :
```
ai:list
```

### Afficher diverses informations sur le contexte actuel :
```
ai:conf
```

### Supprimer tout les messages du contexte :
```
ai:stop
```

### Supprimer tout les messages du contexte et sélectionner le modèle par default :
```
ai:reset
```

### Afficher cette page d'aide :
```
ai:help
```

**Comment by CtrlAiDel** [comment]
C'est vraiment super comme implémentation. Merci beaucoup.

Je comprends pour la liste de user.

Merci beaucoup.
